src/yardstick/cli/config.py

```diff
@@ -45,6 +46,8 @@ class ScanMatrix:
     images: list[str] = field(default_factory=list)
     tools: list[Tool] = field(default_factory=list)
 
+    DIGEST_REGEX = re.compile(r"(?P<digest>sha256:[a-fA-F0-9]{64})")
```

Comment:
I think `sha512` is also valid [per OCI spec](https://github.com/opencontainers/image-spec/blob/main/descriptor.md#digests), and maybe other hashes are considered okay if they're implementation specific? this is probably fine, I don't fully understand all the uses

Reply:
The only use of this code I'm adding is to prevent people from committing images to yardstick configs that are missing any piece that yardstick uses to compute the path (host, path, digest).

I think we can fix yardstick the first time there's a `sha512` image we can to add.